src/main/java/org/spongepowered/common/event/damage/DamageEventHandler.java

```diff
@@ -227,78 +227,80 @@ public static EquipmentType resolveEquipment(int slot) {
     private static double enchantmentDamageTracked;
 
     public static Optional<List<DamageFunction>> createEnchantmentModifiers(EntityLivingBase entityLivingBase, DamageSource damageSource) {
-        Iterable<net.minecraft.item.ItemStack> inventory = entityLivingBase.getArmorInventoryList();
-        if (EnchantmentHelper.getEnchantmentModifierDamage(Lists.newArrayList(entityLivingBase.getArmorInventoryList()), damageSource) == 0) {
-            return Optional.empty();
-        }
-        List<DamageFunction> modifiers = new ArrayList<>();
-        boolean first = true;
-        int totalModifier = 0;
-        for (net.minecraft.item.ItemStack itemStack : inventory) {
-            if (itemStack.isEmpty()) {
-                continue;
-            }
-            NBTTagList enchantmentList = itemStack.getEnchantmentTagList();
-            if (enchantmentList == null) {
-                continue;
+        if (!damageSource.isDamageAbsolute()) {
```

Comment:
No idea why this sections looks like it has huge differences in different places. This was the only change I made, obviously indenting everything inside.